sui_core/src/authority/authority_store.rs

```diff
@@ -240,7 +243,7 @@ impl<const ALL_OBJ_VER: bool, S: Eq + Serialize + for<'de> Deserialize<'de>>
         Ok(self
             .objects
             .iter()
-            .skip_to(&ObjectID::ZERO)?
+            .skip_to(&ObjectKey::ZERO)?
```

Comment:
Not your fault but I feel this method should be called `objects_table_is_empty` because DB could potentially have other data but no objects.